### ISSUE TITLE
[BUG] Missing requirement `packaging` in 3.11.1

### ISSUE THREAD

**Issue by petrklus** [django-cms/django-cms#7459]
## Description

When upgrading to Django CMS 3.11.1, the following error is showing:

```Step 9/9 : RUN DJANGO_MODE=build python manage.py collectstatic --noinput
 ---> Running in 67cb7a787162
Traceback (most recent call last):
  File "/app/manage.py", line 7, in <module>
    startup.manage(path=os.path.dirname(os.path.abspath(__file__)))
  File "/usr/local/lib/python3.9/site-packages/aldryn_django/startup.py", line 11, in manage
    utility.execute()
  File "/usr/local/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
    django.setup()
  File "/usr/local/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/usr/local/lib/python3.9/site-packages/django/apps/registry.py", line 114, in populate
    app_config.import_models()
  File "/usr/local/lib/python3.9/site-packages/django/apps/config.py", line 301, in import_models
    self.models_module = import_module(models_module_name)
  File "/usr/local/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/usr/local/lib/python3.9/site-packages/cms/models/__init__.py", line 4, in <module>
    from .placeholdermodel import *  # nopyflakes
  File "/usr/local/lib/python3.9/site-packages/cms/models/placeholdermodel.py", line 14, in <module>
    from cms.utils import get_language_from_request, get_site_id, permissions
  File "/usr/local/lib/python3.9/site-packages/cms/utils/permissions.py", line 13, in <module>
    from cms.utils.compat.dj import available_attrs
  File "/usr/local/lib/python3.9/site-packages/cms/utils/compat/__init__.py", line 4, in <module>
    from packaging.version import Version
ModuleNotFoundError: No module named 'packaging'
```

The error refers here: https://github.com/django-cms/django-cms/blob/3.11.1/cms/utils/compat/__init__.py#L4

## Steps to reproduce

1. Install Django CMS 3.11.1
2. Run `collectstatic`

## Additional information (CMS/Python/Django versions)

Python 3.9, Django 3.2

**Comment by marksweb** [comment]
Damn. I guess I didn't catch this because I've already got other packages which install this. 🤦‍♂️

**Comment by marksweb** [comment]
Just tracking down changes.

@fsbraun added the requirement to v4 here; https://github.com/django-cms/django-cms/commit/d5f12c9e1560b568687c96de21ae134dc1498799

I'm not sure right now if we want to cherry pick that into develop, or approach this another way.

**Comment by fsbraun** [comment]
I just opened #7462 to fix this. Still, I do not understand why this did not pop up in test runs. If I do `pip-compile` any of the test requirements it does not show `packaging` as a requirement.

**Comment by marksweb** [comment]
@fsbraun how are you running `pip-compile`?

**Comment by fsbraun** [comment]
I just ran `pip-compile test_requirements/django-3.2.txt -o /tmp/something` to inspect the output.

**Comment by gopackgo90** [comment]
If you look at https://github.com/django-cms/django-cms/actions/runs/3748867775/jobs/6366698326 for example, installing `pytest` eventually brings in `packaging`.

**Comment by fsbraun** [comment]
@gopackgo90 Thanks for clarifying this!

**Comment by fsbraun** [comment]
Fixed by #7462
